test/e2e/flask/solana/solana-networks.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { Suite } from 'mocha';
+import { switchToNetworkFlow } from '../../page-objects/flows/network.flow';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import AccountListPage from '../../page-objects/pages/account-list-page';
+import { withSolanaAccountSnap } from './common-solana';
+
+describe('Solana network', function (this: Suite) {
+  it.skip('keeps the network selector enabled when the Solana network is selected', async function () {
```

Comment:
Is it intended to `skip` this one?

```suggestion
  it('keeps the network selector enabled when the Solana network is selected', async function () {
```